src/utils/get-sign-in-url.ts

```diff
@@ -0,0 +1,14 @@
+import getConfig from 'next/config';
+
+const getSignInURL = (path: string): string => {
```

Comment:
if it makes sense to, can you also incorporate this into `src/components/layout.tsx`? If it's different logic then ignore

Reply:
ah yeah I think I meant to do that and forgot, thanks for pointing it out!